lib/matplotlib/backends/backend_wx.py

```diff
@@ -318,7 +318,7 @@ def draw_text(self, gc, x, y, s, prop, angle, ismath=False, mtext=None):
         if angle == 0.0:
             gfx_ctx.DrawText(s, x, y)
         else:
-            rads = angle / 180.0 * math.pi
+            rads = math.radians(angle)
```

Comment:
`math` only has `radians` and `degrees`, not `deg2rad` and `rad2deg`.  I seems that both names are used in the codebase (even before my earlier PR).